src/AmbientSounds.Uwp/App.xaml

```diff
@@ -15,7 +15,483 @@
                 <Setter Property="Background" Value="Transparent" />
                 <Setter Property="CornerRadius" Value="20" />
             </Style>
+            <Style TargetType="GridViewItem" x:Key="GridViewItemExpanded">
```

Comment:
I believe this entire style is not needed?

Reply:
I needed to remove the focus visual because doing so results in this 
![image](https://user-images.githubusercontent.com/25320332/100431598-3c775280-304d-11eb-8f2f-07e419d8c773.png)

Reply:
The following would be enough to do this, as the default Focus visual is not actually a XAML element (andthe expanded template you've used introduces a bunch of extra UI elements into the VisualTree that are never used)

```
<Style TargetType="GridViewItem">
        <Setter Property="UseSystemFocusVisuals" Value="False" />
</Style>
```

But you also *only* want to do this on Xbox, not all the time.

Reply:
The [(SoundGridControl)](https://github.com/jenius-apps/ambie/blob/main/src/AmbientSounds.Uwp/Controls/SoundGridControl.xaml)'s GridView has the ItemContainerStyle as "{StaticResource GridViewItemExpanded}". Which is why I copied the "GridViewItemExpanded" style to the App resources and edited it.

Reply:
I mentioned that only one line of code needs to be moved if the behavior is desired across devices